packages/vanchor/src/token/index.ts

```diff
@@ -0,0 +1,51 @@
+import { Address, Bytes } from '@graphprotocol/graph-ts';
+import { ERC20 } from '../../generated/VAnchor/erc20';
+import { Token } from '../../generated/schema';
+
+export function getTokenSymbol(tokenAddress: Bytes): string {
+    const tokenContract = ERC20.bind(Address.fromBytes(tokenAddress));
+    const symbol = tokenContract.try_symbol();
+
+    if (symbol.reverted) {
+        return 'ETH';
```

Comment:
NOTE: on other chains like Polygon (for example) the native token is not ETH, it is Matic, so we shall handle these kind of stuff in the future.